src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+  const midTerm = 3;
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  const midTermDiscount = 20;
+  const totalCost = pricePerDay * days;
+
+  if (days >= longTerm) {
+    return totalCost - longTermDiscount;
+  }
+
+  if (days >= midTerm) {
+    return totalCost - midTermDiscount;
+  }
+
+  return totalCost;
```

Comment:
Consider using constants for `pricePerDay`, `midTerm`, `longTerm`, `longTermDiscount`, and `midTermDiscount` outside of the function to follow the checklist item 2 and to avoid re-declaring these constants every time the function is called. This will improve the performance slightly by not allocating memory for these constants on each function call.